### PR TITLE
docs: add English version of OpenMLDB Rest_API documentation

### LINKED ISSUE #1495: doc translation (Chinese to English): rest_api
- Translate the Chinese version to English: https://openmldb.ai/docs/zh/v0.4/reference/rest_api.html (you can download as a markdown file from the button at the top-right corner)
- Save the English version to docs/en/rest_api.md

### PR DESCRIPTION
**Pull request by lwq20020127**
close #1495

### CODE REVIEW

**Comment by lwq20020127** [comment]
#1495

File: docs/en/rest_api.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+request body: 
+```
+{
+    "value": [
+    	[v1, v2, v3]
+    ]
+}
+```
+
++ Only one data can be inserted at a time.
```
Comment (lumianph) [inline docs/en/rest_api.md:18]: one data => one record

File: docs/en/rest_api.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+response:
+
+```
+{
+    "code":0,
+    "msg":"ok"
+}
+```
+
+## Real-Time Feature Computation
```
Comment (lumianph) [inline docs/en/rest_api.md:38]: Computation => Extraction

File: docs/en/rest_api.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+```
+{
+    "input": [["row0_value0", "row0_value1", "row0_value2"], ["row1_value0", "row1_value1", "row1_value2"], ...],
+    "need_schema": false
+}
+```
+
++ Multiple rows input are supported which correspond to the array of data.data fields in the returned value.
++ A schema will be returned if `need_schema`  if `true`. Default: false.
```
Comment (lumianph) [inline docs/en/rest_api.md:54]: if `need_schema`  if `true`
=>
if `need_schema`  is `true`

File: docs/en/rest_api.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+request body: 
+
+```
+{
+    "input": [["row0_value0", "row0_value1", "row0_value2"], ["row1_value0", "row1_value1", "row1_value2"], ...],
+    "need_schema": false
+}
+```
+
++ Multiple rows input are supported which correspond to the array of data.data fields in the returned value.
```
Comment (lumianph) [inline docs/en/rest_api.md:53]: Multiple rows of input are supported, whose returned values correspond to the fields in the data.data array.

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/rest_api.md (hunk 4 above)
Comment (lwq20020127) [reply docs/en/rest_api.md:53]: Thank you for your suggestions! Should I close this PR and create a new one?

Comment (lumianph) [reply docs/en/rest_api.md:53]: no. just push more commits here :-)

Comment (lwq20020127) [reply docs/en/rest_api.md:53]: I am sorry that I have deleted the repo I forked before. I am afraid I can not push commit to this PR, so I created a new PR #1526. Thank you!

Comment (lumianph) [reply docs/en/rest_api.md:53]: all right, then i will close this PR

**Pull request closed without merging**